Sources/WalletConnectRouter/Router/Router.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+public struct WalletConnectRouter {
+    public static func goBack(uri: String) {
```

Comment:
what happens if the uri is an empty string for example?
should the function throw?